libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -126,6 +127,9 @@ class AP_Frsky_Telem
     // update flight control mode. The control mode is vehicle type specific
     void update_control_mode(uint8_t mode) { _ap.control_mode = mode; }
 
+    // update whether we're flying (used for Plane)
+    void set_is_flying(bool is_flying) { *_ap.value == is_flying ? (*_ap.value | AP_ISFLYING_FLAG) : (*_ap.value & (~AP_ISFLYING_FLAG)); }
```

Comment:
I had a typo in this PR which I corrected. This line of code is new to this PR (not in master). Now passes sitl...